docs/resources/rule.md

```diff
@@ -72,6 +74,7 @@ resource "jupiterone_rule" "unencrypted_critical_data_stores" {
 - `question_id` (String) Specifies the ID of a question to be used in rule evaluation.
 - `question_name` (String) Specifies the name of a question to be used in rule evaluation.
 - `spec_version` (Number) Rule evaluation specification version in the case of breaking changes.
+- `tags` (List of String) Tags to apply to the rule.
```

Comment:
comma separated?

Reply:
Maybe we could just include an example with multiple tags on line 29?

Reply:
Updated in latest commit.